capstone-backend/app.js

```diff
@@ -1,49 +1,39 @@
-var express = require('express');
-var path = require('path');
-var logger = require('morgan');
-var cookieParser = require('cookie-parser');
-var bodyParser = require('body-parser');
-
-var callback = require('./routes/callback');
-
-var app = express();
-
-// view engine setup
-app.set('views', path.join(__dirname, 'views'));
-app.set('view engine', 'ejs');
-
-// uncomment after placing your favicon in /public
-//app.use(favicon(path.join(__dirname, 'public', 'favicon.ico')));
-app.use(logger('dev'));
-app.use(bodyParser.json());
-app.use(bodyParser.urlencoded({ extended: false }));
-app.use(cookieParser());
-app.use(express.static(path.join(__dirname, 'public')));
-
-// app.use('/', index);
-app.use(express.static(path.join(__dirname, 'client/build')));
-app.use('/callback', callback);
-
-app.get('/*', (req, res) => {
-  res.sendFile(path.join(__dirname, 'client/build', 'index.html'))
+const express = require('express')
+const morgan = require('morgan')
+const cors = require('cors')
+const Parse = require('parse/node')
+const {PARSE_APP_ID, PARSE_JAVASCRIPT_KEY} = require('./config')
+
+const app = express()
+const port = process.env.PORT || 3001
+
+app.use(express.json())
+app.use(morgan("tiny"))
+app.use(cors())
+
+Parse.initialize(PARSE_APP_ID, PARSE_JAVASCRIPT_KEY)
+Parse.serverURL = "https://parseapi.back4app.com"
+
+app.post('/register', async (req, res) => {
+  let user = new Parse.User(req.body)
+
+  try {
+      await user.signUp()
+      res.status(201)    
+      res.send({"user" : user})
```

Comment:
Nit: 
```
res.send({ user })
```
Would also work. This is thanks to es6 "property value shorthand "
https://ui.dev/shorthand-properties